build.gradle.kts

```diff
@@ -10,7 +10,7 @@ plugins {
 }
 
 group = "uk.nhs.hee.tis.trainee"
-version = "0.12.0"
+version = "0.12.1"
```

Comment:
This should be a minor version bump, since it's a functional change and not a bug fix.